libraries/ms-common/src/stm32f0xx/pwm.c

```diff
@@ -128,7 +128,7 @@ StatusCode pwm_set_dc(PWMTimer timer, uint16_t dc) {
 
   uint16_t pulse_width = 0;
   if (dc != 0) {
-    pulse_width = ((s_period_ms[timer] + 1) * dc) / 100 - 1;
+    pulse_width = ((s_period_us[timer] + 1) * dc) / 100 - 1;
```

Comment:
Can you add a check for invalid pulse widths?

Reply:
doesn't it already have a check for this? 
```
else if (pulse_width_ms > s_period_ms[timer]) {
    return status_msg(STATUS_CODE_INVALID_ARGS, "Pulse width must be leq period.");
}
```

Reply:
No, that only check for a pulse width greater than the period.